Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```
@@ -145,7 +145,7 @@
     bigTrigger: true
   - type: UseDelayOnShoot
   - type: UseDelay
-    delay: 0.75 #No script abuse
+    delay: 0.85 #No script abuse
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)